kuma/javascript/src/header/search.jsx

```diff
@@ -50,7 +50,7 @@ export default function Search(props: Props) {
         if (inputfield.current && props.initialQuery) {
             inputfield.current.value = props.initialQuery;
         }
-    }, []);
+    }, [props.initialQuery]);
```

Comment:
This seems like it might be a legitimate actual bug that the linter found here. I don't think it was causing any problem in practice, but this change is a real improvement.